prac_05/Emails.py

```diff
@@ -0,0 +1,34 @@
+"""
+CP1404Practical
+Emails Program
+"""
+
+
+def main():
+    """Create dictionary of emails-to-names."""
+    email_to_name = {}
+    email = input("Email: ")
+    while email != "":
+        name = get_name_from_email(email)
+        answer = input(f"Is your name {name}? (Y/n)")
+        if answer.upper() == "Y" or answer == "":
```

Comment:
This loop could be done using != "Y", which would not need an else statement.

Reply:
Yeah, that does look wrong, just fixed it, thank you XD